packages/css-themes-polyfill/Makefile

```diff
@@ -0,0 +1,26 @@
+VERSION := patch
+GITHUB_API_TOKEN := ""
+
+release:
```

Comment:
with the introduction of `lerna` and `$ lerna publish`, this `make release` is now redundant.